lib/Upcoming_Ride.dart

```diff
@@ -1,277 +1,85 @@
-                    ),
-                  ),
-                  Container(
-                      margin: EdgeInsets.symmetric(horizontal: 15.0),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.start,
-                        children: <Widget>[
-                          Text('From', style: directionStyle)
-                        ],
-                      )
-                  ),
-                  Container(
-                      margin: EdgeInsets.symmetric(horizontal: 15.0),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.start,
-                        children: <Widget>[
-                          Text('Uris Hall', style: infoStyle)
-                        ],
-                      )
-                  ),
-                  Container(
-                      margin: EdgeInsets.symmetric(horizontal: 15.0),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.start,
-                        children: <Widget>[
-                          Text('To', style: directionStyle)
-                        ],
-                      )
-                  ),
-                  Container(
-                      margin: EdgeInsets.symmetric(horizontal: 15.0),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.start,
-                        children: <Widget>[
-                          Text('Cascadilla Hall', style: infoStyle)
-                        ],
-                      )
-                  ),
-                  Container(
-                      margin: EdgeInsets.symmetric(horizontal: 15.0),
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.start,
-                        children: <Widget>[
-                          Text('Ride Confirmed', style: confirmationStyle)
-                        ],
-                      )
-                  ),
-                  Container(
-                    margin: EdgeInsets.symmetric(horizontal: 15.0),
-                    child: Row(
-                      children: <Widget>[
-                        Padding(
-                          padding: const EdgeInsets.only(top: 8.0),
-                          child: Icon(Icons.person_pin, size: 38.0),
-                        ),
-                        SizedBox(width: 5.0),
-                        Column(
-                          crossAxisAlignment: CrossAxisAlignment.start,
-                          children: <Widget>[
-                            Text('Davea Butler', style: infoStyle),
-                            Row(
-                              children: <Widget>[
-                                Icon(Icons.phone, size: 13.0,),
-                                SizedBox(width: 5.0),
-                                GestureDetector(
-                                  onTap: () => UrlLauncher.launch("tel://13232315234"),
```

Comment:
Should the rider be able to call the driver from this card? If so, you should add the UrlLauncher into your code, if not, disregard this comment.